pkg/httpmetrics/metrics.go

```diff
@@ -300,11 +301,14 @@ func instrumentHandlerCounter(counter *prometheus.CounterVec, next http.Handler)
 		}
 
 		next.ServeHTTP(d, r)
+		ua := useragent.Parse(r.UserAgent())
 		counter.With(prometheus.Labels{
-			"method":     r.Method,
-			"code":       strconv.Itoa(d.Status),
-			"ce_type":    r.Header.Get(CeTypeHeader),
-			"user_agent": r.Header.Get("User-Agent"),
+			"method":                     r.Method,
+			"code":                       strconv.Itoa(d.Status),
+			"ce_type":                    r.Header.Get(CeTypeHeader),
+			"user_agent_browser_name":    ua.Name,
+			"user_agent_browser_version": ua.Version,
```

Comment:
do we want to use `VersionNoShort()` rather than the full version string to have better grouping?
https://github.com/mileusna/useragent?tab=readme-ov-file#version-parsing

and similarly for OS, OSVersionNoShort()

Reply:
Yep!  Happy to do that.  Pushing now.